### PR TITLE
Update ServicesExtensions.cs

### PR DESCRIPTION
**Pull request by pingvin1308**
replace string format by index with names

it can be useful for log indexing systems that tries to index values that services passes into the string.

### CODE REVIEW

**Comment by mgravell** [comment]
Fair enough; found the docs: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-7.0#log-message-template - checks out and makes sense

**Comment by mgravell** [comment]
can you update /docs/releasenotes.md too, please? see existing content for format - this would be in the "unreleased" section currently

**Review by mgravell — APPROVED** [review]

File: src/protobuf-net.Grpc.AspNetCore/ServicesExtensions.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public CodeFirstServiceMethodProvider(ILoggerFactory loggerFactory, BinderConfig
             void IServiceMethodProvider<TService>.OnServiceMethodDiscovery(ServiceMethodProviderContext<TService> context)
             {
                 int count = new Binder(_logger).Bind<TService>(context, _binderConfiguration);
-                if (count != 0) _logger.Log(LogLevel.Information, "RPC services being provided by {0}: {1}", typeof(TService), count);
+                if (count != 0) _logger.Log(LogLevel.Information, "RPC services being provided by {service}: {count}", typeof(TService), count);
```
Comment (meirkr) [inline src/protobuf-net.Grpc.AspNetCore/ServicesExtensions.cs:46]: Hi
The convention is to have PascalCase:
[CA1727: Use PascalCase for named placeholders](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1727)
```suggestion
                if (count != 0) _logger.Log(LogLevel.Information, "RPC services being provided by {Service}: {Count}", typeof(TService), count);
```

**Comment by mgravell** [comment]
merging, thanks

**Pull request merged**